pymtl3/passes/yosys/translation/structural/YosysStructuralTranslatorL1.py

```diff
@@ -253,7 +253,8 @@ def rtlir_tr_current_comp( s, comp_id, comp_rtype ):
     s.deq.append( {'attr':[], 'index':[], 's_attr':"", 's_index':""} )
     return ''
 
-  def rtlir_tr_literal_number( s, nbits, value ):
+  def rtlir_tr_literal_number( s, nbits, value, first_called = True ):
```

Comment:
I'll think more about the plan for the yosys backend. Seems like it's becoming even more hacky...